android-sdk/src/grpc/java/com/mobilecoin/lib/network/services/transport/grpc/GRPCTransport.java

```diff
@@ -42,17 +42,17 @@ public GRPCTransport(@NonNull MobileCoinUri uri, @NonNull ClientConfig.Service s
                             uri.getUri().getHost(),
                             uri.getUri().getPort()
                     );
-            if (uri.isTlsEnabled()) {
-                managedChannelBuilder.useTransportSecurity();
-            } else {
-                managedChannelBuilder.usePlaintext();
-            }
             Set<X509Certificate> trustRoots = serviceConfiguration.getTrustRoots();
```

Comment:
SSLSocketFactory is not useful if TLS is disabled, can this go inside this check `if (uri.isTlsEnabled())`

Reply:
Yes, that works too